applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_incompressible_potential_element.cpp

```diff
@@ -209,6 +210,34 @@ namespace Kratos {
       }
     }
 
+    KRATOS_TEST_CASE_IN_SUITE(PingEmbeddedIncompressiblePotentialFlowElementLHS, CompressiblePotentialApplicationFastSuite) {
+      Model this_model;
+      ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+      GenerateEmbeddedElement(model_part);
+      Element::Pointer p_element = model_part.pGetElement(1);
+      const unsigned int number_of_nodes = p_element->GetGeometry().size();
+
+      // Define the distance values
+      std::array<double, 3> level_set{1.0, -1.0, -1.0};
+      for (unsigned int i = 0; i < 3; i++) {
+          p_element->GetGeometry()[i].FastGetSolutionStepValue(GEOMETRY_DISTANCE) = level_set[i];
+      }
+
+      const std::array<double, 3> potential{1.0, 20.0, 50.0};
+
+      Matrix LHS_finite_diference = ZeroMatrix(number_of_nodes, number_of_nodes);
+      Matrix LHS_analytical = ZeroMatrix(number_of_nodes, number_of_nodes);
+
+      PotentialFlowTestUtilities::ComputeElementalSensitivities<3>(model_part, LHS_finite_diference, LHS_analytical, potential);
+
+      for (unsigned int i = 0; i < LHS_finite_diference.size1(); i++) {
+          for (unsigned int j = 0; j < LHS_finite_diference.size2(); j++) {
+              KRATOS_CHECK_NEAR(LHS_finite_diference(i,j), LHS_analytical(i,j), 1e-10);
+          }
+      }
```

Comment:
you can use the matrix compare macro

Reply:
Ok, I took the chance to update all tests with this